src/others/xml/parsers/lv_xml_label_parser.c

```diff
@@ -58,12 +58,25 @@ void lv_xml_label_apply(lv_xml_parser_state_t * state, const char ** attrs)
         if(lv_streq("text", name)) lv_label_set_text(item, value);
         if(lv_streq("long_mode", name)) lv_label_set_long_mode(item, long_mode_text_to_enum_value(value));
         if(lv_streq("bind_text", name)) {
-            lv_subject_t * subject = lv_xml_get_subject(&state->ctx, value);
-            if(subject) {
-                lv_label_bind_text(item, subject, NULL);
-            }
-            else {
-                LV_LOG_WARN("Subject \"%s\" doesn't exist in label bind_text", value);
+            char buf[256];
+            lv_strncpy(buf, value, sizeof(buf));
+            char * bufp = buf;
+            char * subject_name = lv_xml_split_str(&bufp, ' ');
+            if(subject_name) {
+                lv_subject_t * subject = lv_xml_get_subject(&state->ctx, subject_name);
+                if(subject) {
+                    char * fmt = bufp; /*The second part is the format text*/
+                    if(fmt[0] == '\0') fmt = NULL;
+                    if(fmt) {
+                        if(fmt[0] == '\'') fmt++;
+                        size_t fmt_len = lv_strlen(fmt);
+                        if(fmt[fmt_len - 1] == '\'') fmt[fmt_len - 1] = '\0';
```

Comment:
```suggestion
                        if(fmt_len != 0 && fmt[fmt_len - 1] == '\'') fmt[fmt_len - 1] = '\0';
```

else `fmt_len - 1` will overflow since we're dealing with `size_t`